packages/editor-ui/src/components/mixins/pinData.ts

```diff
@@ -0,0 +1,18 @@
+import Vue from 'vue';
+import { INodeUi } from "@/Interface";
+import { IDataObject } from "n8n-workflow";
+
+export const pinData = Vue.extend({
+	props: ['name'],
+	computed: {
+		node (): INodeUi | undefined {
+			return this.$store.getters.nodesByName[this.name] as INodeUi | undefined;
+		},
```

Comment:
Does this belong here? Looks more general rather than related specifically to pindata.

Reply:
It doesn't, but it is needed so that typescript doesn't complain in the mixin. Got any suggestion?

Reply:
Found a solution for this.